.buildbot.sh

```diff
@@ -12,13 +12,13 @@ ulimit -d $((1024 * 1024 * 8)) # 8 GiB
     RUST_BACKTRACE=1 ./x.py test --config .buildbot.toml
 
 # Archive the build and put it in /opt
-TARBALL_TOPDIR=ykrustc-stage2
+TARBALL_TOPDIR=ykrustc-stage2 # To match [install] in the build config.
 TARBALL_NAME=ykrustc-stage2-latest.tar.bz2
 SNAP_DIR=/opt/ykrustc-bin-snapshots
 
-cd build/x86_64-unknown-linux-gnu
-ln -sf stage2 ${TARBALL_TOPDIR}
+mkdir -p ${TARBALL_TOPDIR} # Or you get "cannot canonicalize..."
+env RUST_BACKTRACE=1 ./x.py install --config .buildbot.toml
```

Comment:
Do we need the `RUST_BACKTRACE` thing on? IIRC, it does have some sort of performance consequence.

Reply:
I found a post where someone claims it does, but no-one knows why.

I'd like to have backtraces if possible, because otherwise I may have to build again to know what went wrong.